third_party/egeria/usage/example_transactions.csv.json

```diff
@@ -2,7 +2,7 @@
     "class": "NewCSVFileAssetRequestBody",
     "displayName": "transactions.csv",
     "description": "Synthetic Financial Dataset For Fraud Detection",
-    "fullPath": "{\"data_store\":\"S3\", \"bucket\":\"mybucket\", \"endpoint\":\"s3.eu-de.cloud-object-storage.appdomain.cloud\", \"object_key\":\"transactions.csv\", \"region\":\"eu\", \"data_location\": \"US\"}",
+    "fullPath": "{\"data_store\":\"S3\", \"bucket\":\"mybucket\", \"endpoint\":\"s3.eu-de.cloud-object-storage.appdomain.cloud\", \"object_key\":\"transactions.csv\", \"region\":\"eu\", \"data_location\": \"US\", \"credentials-secret-ref\": 'default/my-secret'}",
```

Comment:
hi @rohithdv  when posting the example_transactions.csv.json to egeria I get error back related to the format of the new field. To simplify things I suggest to change it as follows and it worked ok:

"fullPath": "{\"data_store\":\"S3\", \"bucket\":\"notebook-sample\", \"endpoint\":\"s3.eu-gb.cloud-object-storage.appdomain.cloud\", \"object_key\":\"data.csv\", \"region\":\"eu\", \"data_location\": \"Turkey\",                       \"dataset_creds_secret_name\": \"dataset-creds\", \"dataset_creds_secret_namespace\": \"default\"}"

I appreciate if you could also change the egeria connector code to use dataset_creds_secret_name and  dataset_creds_secret_namespace fields. Thanks